lib/json_elixir/process.ex

```diff
@@ -0,0 +1,83 @@
+defmodule JsonElixir.Process do
+
+  @invalid_input "expected list or bit string got: "
+
+  def to_html!(json) when
```

Comment:
What about to add @spec's?

Reply:
Do we need them?

Reply:
You probably right, it is to small. I think it is not necessary. But can you add some '@moduledoc'

Reply:
I already wrote that in `lib/json_elixir.ex`.